acceptance/acceptance_test.go

```diff
@@ -51,12 +52,17 @@ func TestAccept(t *testing.T) {
 	t.Setenv("DATABRICKS_HOST", fmt.Sprintf("http://127.0.0.1:%d", server.Port))
 	t.Setenv("DATABRICKS_TOKEN", "dapi1234")
 
-	homeDir := t.TempDir()
+	tempHomeDir := t.TempDir()
 	// Do not read user's ~/.databrickscfg
-	t.Setenv(env.HomeEnvVar(), homeDir)
+	t.Setenv(env.HomeEnvVar(), tempHomeDir)
+
+	// Prevent CLI from downloading terraform in each test:
+	t.Setenv("DATABRICKS_TF_EXEC_PATH", tempHomeDir)
 
 	repls := testdiff.ReplacementsContext{}
 	repls.Set(execPath, "$CLI")
+	repls.Set(toJson(t, tempHomeDir), "$TMPHOME")
```

Comment:
Why both `toJson` and verbatim values?

Reply:
Verbatim does not work in JSON output and vice versa.

If verbatim strings contains \\ then you need both replacements if you want replacement to work in json and non-json context.

We don't need non-json here, I added it just in case. However, I think it's best to do that everywhere so you don't think about it -- https://github.com/databricks/cli/pull/2142